api/src/main/java/org/eclipse/microprofile/jwt/algorithm/ContentEncryptionAlgorithm.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
```

Comment:
Years are incorrect

Reply:
What should we put there, `2020-2021` ?

Reply:
For new files, it should be just 2020. (no fork) 

For updated files, the 'end' year should be updated.